### PR TITLE
Tutorial updates and repo cleanup

### PR DESCRIPTION
**Pull request by rabinowitz**

### CODE REVIEW

File: ros-tutorials/lesson-1-basics.md
Comment (ShaanGondalia) [inline ros-tutorials/lesson-1-basics.md:31]: Maybe we should clarify that you don't need to run docker-compose again, just the SSH command? This line is clear to me but might be confusing to someone who just learned about containers and SSH.

File: ros-tutorials/lesson-3-write-a-node.md
Comment (ShaanGondalia) [inline ros-tutorials/lesson-3-write-a-node.md:29]: How does what work? Maybe explicitly say that code changes on your local computer will reflect in the container.

File: ros-tutorials/lesson-3-write-a-node.md
Comment (ShaanGondalia) [inline ros-tutorials/lesson-3-write-a-node.md:35]: These commands aren't needed earlier in the lesson to run turtlesim right?

Comment (rabinowitz) [reply ros-tutorials/lesson-3-write-a-node.md:35]: Nope! I tell them to source ROS directly in the notes, and I've tested it myself. Good call to check though.

File: ros-tutorials/lesson-1-basics.md
Comment (rabinowitz) [reply ros-tutorials/lesson-1-basics.md]: Good call, added clarification.

File: ros-tutorials/lesson-3-write-a-node.md
Comment (rabinowitz) [reply ros-tutorials/lesson-3-write-a-node.md]: Thanks, clarified this.

**Comment by ShaanGondalia** [comment]
LGTM, will merge after someone else reads over it.

File: ros-tutorials/lesson-0-introduction.md
Comment (muthuArivoli) [inline ros-tutorials/lesson-0-introduction.md:20]: I don't think this warning is necessary - with WSL 2, everything is smooth. I don't want someone wasting their time upgrading their system, and that might break stuff since Windows doesn't do well with upgrades (lol I don't want someone suing us if all their data is deleted or something).

You however may want to mention updating their system to the May 2020 update, I think its Windows version 1903 or something. Then again, updating is probably as bad as upgrading, so not sure which is lesser of two evils.

File: ros-tutorials/lesson-0-introduction.md
Comment (muthuArivoli) [inline ros-tutorials/lesson-0-introduction.md:28]: This could just be my computer, since I've messed around with the system path, but Git doesn't work in cmd prompt, it only works in git bash. If it works on you computer then this should be fine, but otherwise git bash might be recommended. Also, not sure if you want to tell them to move to the directory they want to clone in, instead of just cloning in their home directory (I know if you install Pspice, your home directory gets changed to the Pspice install location). You could also tell them to open the directory in terminal using right click or something like that.

File: ros-tutorials/lesson-0-introduction.md
Comment (muthuArivoli) [inline ros-tutorials/lesson-0-introduction.md:45]: 90% sure that `cd documentation/ros-tutorials` also works for Windows, so you unify these.

File: ros-tutorials/lesson-1-basics.md
Comment (muthuArivoli) [inline ros-tutorials/lesson-1-basics.md:27]: I don't think you need sudo anymore with root access. Also I prefer commands not to be chained with ands, but it's not a big deal if it is.

File: ros-tutorials/lesson-1-basics.md
Comment (muthuArivoli) [inline ros-tutorials/lesson-1-basics.md:4]: nit: you know how much I love **reading** those videos.

File: ros-tutorials/lesson-3-write-a-node.md
Comment (muthuArivoli) [inline ros-tutorials/lesson-3-write-a-node.md:16]: Program the node to do what? I guess that's in the second step, but I kind of found that unclear.

File: unsorted/mobaxterm
Comment (muthuArivoli) [inline unsorted/mobaxterm:1]: Might want to make sure this is notated down somewhere - I know its in the intro tutorial, but if someone hasn't done the intro tutorial, they will face this problem when using our codebase.

**Review by muthuArivoli — CHANGES_REQUESTED** [review]
Looking good, mostly nits. My main concern right now is that the files that were already merged in #15 are showing up here, which is probably not a good sign. You might have to rebase or do something like that.

File: unsorted/mobaxterm
Comment (rabinowitz) [reply unsorted/mobaxterm:1]: This will also be noted in the new README in the main repo.

File: ros-tutorials/lesson-3-write-a-node.md
Comment (rabinowitz) [reply ros-tutorials/lesson-3-write-a-node.md:16]: Good call, will clarify.

File: ros-tutorials/lesson-1-basics.md
Comment (rabinowitz) [reply ros-tutorials/lesson-1-basics.md:4]: Lol I was annoyed about that too but left it to keep it shorter. But since it bothered you, too, I added two words to make it make sense.

File: ros-tutorials/lesson-0-introduction.md
Comment (rabinowitz) [reply ros-tutorials/lesson-0-introduction.md:45]: You're right, just tested again. Not sure what I was doing differently last time I tested.

File: ros-tutorials/lesson-1-basics.md
Comment (rabinowitz) [reply ros-tutorials/lesson-1-basics.md:27]: Good points on both, will incorporate. Also will change to apt, as that is recommended now.

File: ros-tutorials/lesson-0-introduction.md
Comment (rabinowitz) [reply ros-tutorials/lesson-0-introduction.md:28]: Git works in command prompt if you've added it to your path, which is an option during installation (and seems to be the default, but I can make an explicit note if you'd like, just let me know.)

About the cloning directory, I figured I'd leave it up to them. This is just the documentation repo anyway.

File: ros-tutorials/lesson-0-introduction.md
Comment (rabinowitz) [reply ros-tutorials/lesson-0-introduction.md:16]: I think it actually does mention it. [This page](https://docs.docker.com/docker-for-windows/install-windows-home/), the one linked in the lesson here, has it in the first section "What to know before you install" > "System Requirements", a whole bit about updating Windows and using WSL 2.

File: ros-tutorials/lesson-0-introduction.md
Comment (rabinowitz) [reply ros-tutorials/lesson-0-introduction.md:20]: Sounds good. I haven't used WSL 2, so I was just going off what we've talked about previously. The system update is probably easier than upgrading editions through Duke just for this, so I'll cut out this part entirely.

File: ros-tutorials/lesson-0-introduction.md
Comment (rabinowitz) [reply ros-tutorials/lesson-0-introduction.md:7]: Valid concern, but it's actually one of the easiest to read that I've seen, and isn't that long, just has extra words and explanation but only covers basic commands. I think I personally could do it in 5-7 minutes, so hopefully someone new could do it in about 15. It's also almost entirely important and relevant to what we do, vs. a lot of other worse command line tutorials.

File: ros-tutorials/docker-compose.yml
Comment (rabinowitz) [reply ros-tutorials/docker-compose.yml:10]: Sure, I can change it to 9090 in case you want to use that. Can't hurt.

File: ros-tutorials/catkin_ws/src/tutorial/CMakeLists.txt
Comment (rabinowitz) [reply ros-tutorials/catkin_ws/src/tutorial/CMakeLists.txt:1]: Done. I'm always a fan of simplifying things. :)

File: ros-tutorials/catkin_ws/src/tutorial/package.xml
Comment (rabinowitz) [reply ros-tutorials/catkin_ws/src/tutorial/package.xml:1]: Done here as well.

File: intro-project/README.md
Comment (rabinowitz) [reply intro-project/README.md:1]: About to push all my changes, which should include a correct rebase on master.

**Review by muthuArivoli — APPROVED** [review]
LGTM!

**Pull request merged**
